pyirf/io/gadf.py

```diff
@@ -85,8 +85,8 @@ def create_aeff2d_hdu(
 @u.quantity_input(
     psf=u.sr ** -1,
     true_energy_bins=u.TeV,
-    fov_offset_bins=u.deg,
     source_offset_bins=u.deg,
+    fov_offset_bins=u.deg,
```

Comment:
I think it was right before (is the order non-commutable here?)

PSF requires axes (energy, offset, rad) where `offset` is FoV offset and `rad` is source offset (like you wrote in the docstring)

Reply:
It is just moved to have the order be the same as the argument order. No change in behaviour.